vllm/model_executor/layers/lightning_attn.py

```diff
@@ -477,7 +477,7 @@ def lightning_attention(q, k, v, ed, block_size=256, kv_history=None):
         ed = ed.view(1, -1, 1, 1)
 
     # Split the computation into chunks for better parallelism
-    m = 128 if d >= 128 else 64
+    m = 128 if d >= 128 else 32
```

Comment:
This change was needed to run this tiny model, need to revert it once tested on big model.